src/main/java/com/batix/rundeck/core/AnsibleRunner.java

```diff
@@ -346,12 +346,9 @@ public int run() throws Exception {
     }
 
     if (debug == Boolean.TRUE) {
-      procArgs.add("-vvvv");
-    } else {
-      procArgs.add("-v");

```

Comment:
removed the default -v allowing ansible log output to be normal. -v can always be added by the playbook creator in Rundeck as an option.